packages/react-instantsearch/src/widgets/__tests__/RelatedProducts.test.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { createSearchClient } from '@instantsearch/mocks';
+import { InstantSearchTestWrapper } from '@instantsearch/testutils';
+import { render, waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { RelatedProducts } from '../RelatedProducts';
+
+describe('RelatedProducts', () => {
+  test('renders with translations', async () => {
+    const client = createSearchClient();
```

Comment:
This should probably be using a client with mock recommendations if we want to test translations for the title.

Reply:
Oops, my bad, I overlooked this test.